model.js

```diff
@@ -41,33 +39,44 @@ exports.fetchArticleById = (inputId) => {
     A.votes,
     A.article_img_url 
     FROM articles A
-    WHERE article_id = $1`
+    WHERE article_id = $1`;
 
-    if (inputId > testData.articleData.length) {
-        return Promise.reject({status: 400, msg: 'Bad Request'})}
+  if (inputId > testData.articleData.length) {
+    return Promise.reject({ status: 400, msg: "Bad Request" });
+  }
 
-    return db.query(queryString, [inputId])
-    .then((article) => {
-        return article.rows[0]
-    })
-}
+  return db.query(queryString, [inputId]).then((article) => {
+    return article.rows[0];
+  });
+};
 
 exports.fetchComments = (inputId) => {
-    const queryString = `SELECT
+  const queryString = `SELECT
     A.comment_id,
     A.votes,
     A.created_at,
     A.author,
     A.body,
     A.article_id
     FROM comments A
-    WHERE article_id=$1 ORDER BY created_at DESC`
-    if (inputId > testData.articleData.length) {
-        return Promise.reject({status: 400, msg: 'Bad Request'})}
-    return db.query(queryString, [inputId])
-    .then((comments) => {
-        return comments.rows
-    })
-}
+    WHERE article_id=$1 ORDER BY created_at DESC`;
+  if (inputId > testData.articleData.length) {
```

Comment:
this code is not relevant to your app in development and production environments; you're going to need a different, dynamic check